src/components/StakingPage/StakingPanel/LockPeriodOptions.tsx/index.tsx

```diff
@@ -29,88 +29,153 @@ const LockPeriod = memo((props: ILockPeriodProps) => {
   );
 });
 
-const mockingYearsPeriod = [4, 3, 2, 1, 0];
-const mockingMonthsPeriod = [9, 3, 1, 0];
+const yearsPeriod = [4, 3, 2, 1, 0];
+const monthsPeriod = [9, 6, 3, 0];
 
+const monthlyTimeStampTransfer = (month: number) => month * 30 * 24 * 60 * 60;
+const yearlyTimeStampTransfer = (years: number) => years * 8766 * 60 * 60;
+
+const periodsGroup = [
+  [4, [0]],
+  [3, [0, 3, 6, 9]],
+  [2, [0, 3, 6, 9]],
+  [1, [0, 3, 6, 9]],
+  [0, [3, 6, 9]],
+] as [number, number[]][];
+
+const timestampGroups = periodsGroup

```

Comment:
This will keep the same for every terms, I think it could be a constants or Sigleton